### ISSUE TITLE
Allow easier access to header / body font families for overrides

### ISSUE THREAD

**Issue by jaredly** [KyleAMathews/typography.js#115]
Looks like the current best way to get the headerFontFamily into something that's not an `hX` or similar is this, taken from [gatsby](https://github.com/gatsbyjs/gatsby/blob/master/www/src/layouts/index.js#L105):
```
            css={{

              fontFamily: typography.options.headerFontFamily.join(`,`),
            }}
```

it would be great to provide a helper method that would make it feel first-class

**Comment by KyleAMathews** [comment]
good idea! Now that Gatsby is at a 1.0, planning on circling back to Typography.js sooner than later to get it over the hump to a 1.0.
